core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreManager.java

```diff
@@ -487,13 +606,39 @@ public X509Certificate getDefaultPrimaryCertificate() throws Exception {
                 "available only for the super tenant.");
     }
 
+    /**
+     * This method is used to get public certificates of tenant keystores
+     *
+     * @return Public certificate of a given tenant keystore
+     * @throws Exception Permission denied for accessing primary key store
+     */
+    public Certificate getTenantCertificate(String keyStoreName, String alias) throws Exception {
+
+        return getTenantKeyStore(keyStoreName).getCertificate(alias);
+    }
+
+    /**
+     * This method is used to get public certificates of custom keystores
+     *
+     * @return Public certificate of a given custom keystore
+     * @throws Exception Permission denied for accessing primary key store
+     */
```

Comment:
```suggestion
    /**
     * This method is used to get public certificates of custom keystores.
     *
     * @return Public certificate of a given custom keystore.
     * @throws Exception Permission denied for accessing the primary key store.
     */
```

Please check other similar places

Reply:
Fixed in https://github.com/wso2/carbon-kernel/commit/a0533c76776fd427f270477373ea6951ce2f147f